man/networkd.conf.xml

```diff
@@ -256,6 +256,17 @@ DUIDRawData=00:00:ab:11:f9:2a:c2:77:29:f9:5c:00</programlisting>
           <xi:include href="version-info.xml" xpointer="v230"/>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>UseDomains=</varname></term>
+          <listitem><para>Specifies the default value for <varname>UseDomains=</varname>.
```

Comment:
`default value for <varname>UseDomains=</varname>` -> `default value for per-network <varname>UseDomains=</varname>` is better?